docs/errors.rst

```diff
@@ -79,6 +81,10 @@ raised or returned, depending on which method or function is used.
         `instance`\). The deque can be empty if the error happened
         at the root of the instance.
 
+    .. attribute:: json_path
+
+        A JSON path to the offending element within the instance.
```

Comment:
Can you add a link to the `JSON path` specification here?

Reply:
Done
There is no official spec though but the website I've linked to is the closest to it.
It's the original document that is usually linked to (see http://jsonpath.com/ link for example)